waku/node/v2/wakunode2.nim

```diff
@@ -192,6 +192,12 @@ proc start*(node: WakuNode, conf: WakuNodeConf) {.async.} =
   node.switch.mount(wakuRelayProto)
   wakuRelayProto.started = true
 
+  let storeProto = WakuStore.init()
+  node.switch.mount(storeProto)
+
+  # @TODO THIS DOES NOT WORK, HOW DO I GET ACCESS TO `WakuRelay`
+  # wakuRelayProto.filters["store"] = storeProto.filter()
```

Comment:
This should be a tell for a big code smell and why the way "filters" are implemented might not be right.